packages/next/errors.json

```diff
@@ -632,5 +632,6 @@
   "631": "Invariant (SlowModuleDetectionPlugin): Circular dependency detected in module graph. This is a Next.js internal bug.",
   "632": "Invariant (SlowModuleDetectionPlugin): Module is missing a required debugId. This is a Next.js internal bug.",
   "633": "Dynamic route not found",
-  "634": "Route %s used \"searchParams\" inside \"use cache\". Accessing Dynamic data sources inside a cache scope is not supported. If you need this data inside a cached function use \"searchParams\" outside of the cached function and pass the required dynamic data in as an argument. See more info here: https://nextjs.org/docs/messages/next-request-in-use-cache"
+  "634": "Route %s used \"searchParams\" inside \"use cache\". Accessing Dynamic data sources inside a cache scope is not supported. If you need this data inside a cached function use \"searchParams\" outside of the cached function and pass the required dynamic data in as an argument. See more info here: https://nextjs.org/docs/messages/next-request-in-use-cache",
+  "635": "%s: Invalid source map. Only conformant source maps can be used to find the original code."
```

Comment:
It's pretty jargon-y. The error cause is likely also not sufficient since the invalid sourcemap may have been produced by our bundler due to invalid input. But we also never have access to the source mapping URL. We only ever get access to the payload if it fails when constructing the source map consumer so we could theoretically dump the contents somewhere and link to that place. This feels excessive for such a rare issue that is in most cases caused by Next.js bugs (e.g. using `turbopack://[project]` as URLs so I feel comfortable with this error message.

 Open to suggestions how to improve the wording though.